suite/tests/api/dis-a64.txt

```diff
@@ -1304,16 +1304,21 @@ d5033f5f : clrex                          : clrex  $0x0f
 d5033f9f : dsb    sy                      : dsb    $0x0f
 d5033fbf : dmb    sy                      : dmb    $0x0f
 d5033fdf : isb                            : isb    $0x0f
-d5080000 : sys    #0, C0, C0, #0, x0      : sys    $0x0000 %x0
-d50fffff : sys    #7, C15, C15, #7        : sys    $0x3fff %xzr
+d5080000 : sys    #0, C0, C0, #0, x0      : sys    $0x00 $0x00 $0x00 $0x00 %x0
+d50fffff : sys    #7, C15, C15, #7        : sys    $0x07 $0x0f $0x0f $0x07 %xzr
+d50b7420 : sys    #3,  C7,  C4, #1, x0    : sys    $0x03 $0x07 $0x04 $0x01 %x0
+d50b7a21 : sys    #3,  C7,  C10, #1, x1   : sys    $0x03 $0x07 $0x0a $0x01 %x1
+d50b7b21 : sys    #3,  C7,  C11, #1, x1   : sys    $0x03 $0x07 $0x0b $0x01 %x1
+d50b7e21 : sys    #3,  C7,  C14, #1, x1   : sys    $0x03 $0x07 $0x0e $0x01 %x1
+d50b7521 : sys    #3,  C7,  C5,  #1, x1   : sys    $0x03 $0x07 $0x05 $0x01 %x1
 d5100000 : msr    s2_0_c0_c0_0, x0        : msr    %x0 $0x0000
 d51b4201 : msr    nzcv, x1                : msr    %x1 -> %nzcv
 d51b4402 : msr    fpcr, x2                : msr    %x2 -> %fpcr
 d51b4423 : msr    fpsr, x3                : msr    %x3 -> %fpsr
 d51bd044 : msr    tpidr_el0, x4           : msr    %x4 -> %tpidr_el0
 d51fffff : msr    s3_7_c15_c15_7, xzr     : msr    %xzr $0x7fff
-d5280000 : sysl   x0, #0, C0, C0, #0      : sys    $0x0000 -> %x0
-d52fffff : sysl   xzr, #7, C15, C15, #7   : sys    $0x3fff -> %xzr
+d5280000 : sysl   x0, #0, C0, C0, #0      : sysl   $0x00 $0x00 $0x00 $0x00 -> %x0
+d52fffff : sysl   xzr, #7, C15, C15, #7   : sysl   $0x07 $0x0f $0x0f $0x07 -> %xzr
```

Comment:
Whichever of these is `dc`: I guess we leave the preferred-alias issue to #4382.  Alternatively, for some aliases, we could have actually-separate opcodes (see above as well).